docs/sources/administration/data-source-management/teamlbac/create-teamlbac-rules/index.md

```diff
@@ -0,0 +1,106 @@
+---
+description: Learn how to create Team LBAC rules for the Loki data source.
+keywords:
+  - loki
+  - lbac
+  - team
+labels:
+  products:
+    - enterprise
+    - cloud
+title: Create Team LBAC rules for the Loki data source
+weight: 250
+---
+
+# Create Team LBAC rules for the Loki data source
+
+Team LBAC is available on Cloud for data sources created with basic authentication. Any managed Loki data source can **NOT** be configured with Team LBAC rules.
+
+## Before you begin
+
+- Be sure that you have admin data source permissions for Grafana.
+- Be sure that you have a team setup in Grafana.
+
+### Create a Team LBAC Rule for a team
+
+1. Navigate to your Loki datasource
+1. Navigate to the permissions tab
+   - Here, you'll find the Team LBAC rules section.
+1. Add a Team LBAC Rule
+   - Add a new rule for the team in the Team LBAC rules section.
+1. Define Label Selector for the Rule
+   - Add a label selector to the rule. Refer to Loki query documentation for guidance on the types of log selections you can specify.
+
+#### Best practices
+
+We recommend you only add `query` permissions for teams that should use the data source and only `Admin` have `Admin` permissions.
+
+We recommend for a first setup, setting up as few rules as possible for each team and make them additive for simplicity.
```

Comment:
> and make them additive

What does this mean?

Reply:
`additive` - namespace="dev|prod" making them select the logs they would like to view, rather than negating the logs they would like to view.

@alexanderzobnin do you think it is too much of a recommendation?

Reply:
I just think it's hard to understand